test-butler-app-physical-devices/src/main/java/com/linkedin/android/testbutler/shell/ShellButlerService.java

```diff
@@ -87,6 +89,20 @@ public boolean setGsmState(boolean enabled) throws RemoteException {
         public boolean grantPermission(String packageName, String permission) throws RemoteException {
             return permissionGranter.grantPermission(packageName, permission);
         }
+
+        @Override
+        public boolean setAccessibilityServiceState(boolean enabled) throws RemoteException {
+            return accessibilityServiceEnabler.setAccessibilityServiceEnabled(enabled);
+        }
+
+        @Override
+        public void onDestroy() {
+            super.onDestroy();
+            // Turn the accessibility service off it we enabled it
+            try {
+                accessibilityServiceEnabler.setAccessibilityServiceEnabled(false);
+            } catch (RemoteException ignored) { }
```

Comment:
See comments below -- consider putting enabler in ButlerApiStubBase to avoid code duplication.

Reply:
Same as above.